client/app/scripts/superdesk-monitoring/monitoring.js

```diff
@@ -459,13 +459,17 @@
                 function getType(item) {
                     if (item.state === 'spiked') {
                         return 'spike';
-                    }
-
-                    if (item.state === 'ingested') {
+                    } else if (item.state === 'ingested') {
                         return 'ingest';
-                    }
+                    } else {
+                        var isPublished = _.contains(['published', 'killed', 'scheduled', 'corrected'], item.state);
 
-                    return 'archive';
+                        if (isPublished === false || (isPublished === true && item.allow_post_publish_actions === true)) {
+                            return 'archive';
+                        } else if (isPublished === true && item.allow_post_publish_actions === false) {
```

Comment:
The explicit  comparisons to `true` and `false` are not really needed, because _.contains already returns a boolean, and we can get rid of those (using `isPublished` and `!isPublished` directly).